CONTRIBUTING.md

```diff
@@ -1,181 +1,163 @@
+2. Press the blue button that says “Commit to [branch name].”
 
-[Volunteer to do some translation](https://github.com/thegetty/quire-docs/issues)
+**Push Changes in GitHub**
+1. After you have made all your commits, click on the "Publish Branch" and then “Push Origin” button at the top right.
+2. Now the changes to your project have been successfully saved to your branch on GitHub.
 
 ## Submit Your Contribution
 
-*We manage Quire through GitHub. If you are new to GitHub, we recommend starting with [GitHub Docs](https://docs.github.com/en/free-pro-team@latest/github) to learn by topic, and for a broad overview, [GitHub Guides](https://guides.github.com/). We also encourage you to check out Coding Train’s fantastic video series [Git and Github for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV). GitHub can be accessed through the command line or [GitHub Desktop](https://desktop.github.com/).*
-
-Once you’ve identified your contribution on our GitHub [issues board](https://github.com/thegetty/quire/issues), you’ll next work on it and prepare to submit it to us for review as a pull request. A pull request says, “Hey, I did something useful for you, want to pull it in and merge it into your project?”
+Once you've identified your contribution and submitted changes through GitHub Desktop, you will prepare to submit it to us for review as a pull request. A pull request says, “Hey, I did something useful for you, want to pull it in and merge it into your project?”
 
 A pull request is a chance for Quire’s core team to evaluate the proposed changes made and decide whether to:
 
 - merge them in as is
 - merge them in with changes
 - reject them
 
-To avoid changes being rejected, we recommend you post about proposed changes on our issues board before you start work (which gives the core team and the community the chance to give feedback and raise any concerns ahead of time), and also to submit the pull request while work is still in progress.
-
-A pull request doesn’t have to represent finished work. We always recommend opening a pull request early on, so others can watch or give feedback on your progress. Just mark it as a “WIP” (Work in Progress) in the subject line. As you make new commits to that branch and push them to GitHub, they’ll automatically be added to the open pull request.
+>A pull request doesn’t have to represent finished work. We always recommend opening a pull request early on, so others can watch or give feedback on your progress. When submitting a work in progress pull request, please mark it as a draft by following [these instructions](https://github.blog/2019-02-14-introducing-draft-pull-requests/). As you make new commits to that branch and push them to GitHub, they’ll automatically be added to the open pull request.
 
-This is something like what the development process may look like for you:
+**Create a Pull Request in GitHub**
```

Comment:
See note above, some of this might change slightly for a PR from a Fork versus a Clone